packages/cbioportal-utils/src/model/OncoKB.ts

```diff
@@ -9,6 +9,7 @@ export type Query = {
 
 export interface IOncoKbData {
     indicatorMap: { [id: string]: IndicatorQueryResp } | null;
+    levelTypeSet?: Set<string>;
```

Comment:
I don't think this is supposed to be included?

Reply:
I added it for solving the space issue. Currently it works.